src/kernel.ts

```diff
@@ -547,25 +576,6 @@ class Kernel {
     };
   }
 
-  /**
-   * Close the Websocket.
-   */
-  private _stopChannels(): void {
```

Comment:
Or `disconnect()`, since this isn't taking any action on the session itself. I would believe that either `kill()` or `close()` actually made the server-side session go away, which this doesn't do.